subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -78,6 +100,24 @@ static struct lwm2m_engine_obj_field fields[] = {
 	/* Mark Transport Binding is RO but BOOTSTRAP needs to write it */
 	OBJ_FIELD_DATA(SERVER_TRANSPORT_BINDING_ID, RW, STRING),
 	OBJ_FIELD_EXECUTE(SERVER_REG_UPDATE_TRIGGER_ID),
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+	OBJ_FIELD_EXECUTE(SERVER_BOOTSTRAP_UPDATE_TRIGGER_ID),
+	OBJ_FIELD_DATA(SERVER_APN_LINK_ID, RW_OPT, OBJLNK),
+	OBJ_FIELD_DATA(SERVER_TLS_DTLS_ALERT_CODE_ID, R_OPT, U8),
+	OBJ_FIELD_DATA(SERVER_LAST_BOOTSTRAPPED_ID, R_OPT, TIME),
+	OBJ_FIELD_DATA(SERVER_REGISTRATION_PRIORITY_ORDER_ID, W_OPT, U16),
+	OBJ_FIELD_DATA(SERVER_INITIAL_REGISTRATION_DELAY_TIMER_ID, W_OPT, U16),
+	OBJ_FIELD_DATA(SERVER_REGISRATION_FAILURE_BLOCK_ID, W_OPT, BOOL),
```

Comment:
Same typo here `SERVER_REGISTRATION_FAILURE_BLOCK_ID`

Reply:
Ok, Thanks Jarno. I will fix this.